tests/e2e/cluster_downgrade_test.go

```diff
@@ -96,7 +96,7 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 }
 
 func newCluster(t *testing.T, execPath string, clusterSize int) *e2e.EtcdProcessCluster {
-	epc, err := e2e.NewEtcdProcessCluster(t, &e2e.EtcdProcessClusterConfig{
+	epc, err := e2e.NewEtcdProcessCluster(context.TODO(), t, &e2e.EtcdProcessClusterConfig{
```

Comment:
I see lots of `context.TODO()`, are you going to provide a ctx in separate PR?

Reply:
To avoid to many conflicts I try to do it step by step. In this step I focused on NewCluster in common framework. This leaves a lot of legacy e2e tests that don't use context. I marked them as TODO, so they can be fixed in separate PR.

Reply:
ack